src/Notification.jsx

```diff
@@ -43,3 +43,11 @@ export const ErrorNotification = ({ errorMessage, errorDetail, onDismiss }) => {
         </Alert>
     );
 };
+
+export const WarningNotification = ({ warningMessage, warningDetail }) => {
+    return (
+        <Alert isInline variant='warning' title={warningMessage}>
+            { warningDetail && <p>{warningDetail}</p> }
+        </Alert>
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1270-20230428-110256-01fd85db-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)